Ginger/GingerCore/Drivers/JavaDriverLib/GingerJavaAgent/agent/com/amdocs/ginger/JavaDriver.java

```diff
@@ -1713,6 +1709,60 @@ else if(controlAction.equals("GetControlProperty") && Value.equalsIgnoreCase("VI
 		}
 	}
 
+private TreePath SearchTreeNodes(String locateValue, JTree tree) {
+	TreePath treePath = null;
+	int startNodeNumber = 0;
+	boolean nodeFound = false; 
+	String[] nodes = locateValue.split("/");					
+	for (String node : nodes)
+	{
+		startNodeNumber = (treePath == null ? 0 : (tree).getRowForPath(treePath));
+		(tree).expandRow(startNodeNumber);
+		for(int i = startNodeNumber; i <= ((tree).getRowCount() - 1); i++)
+		{
+			if(nodeFound)
+			{
+				break;
+			}
+			else
+			{
+				treePath = (tree).getNextMatch(node.trim(), i, Position.Bias.Forward);
+				
+				if(treePath != null)
+				{
+					DefaultMutableTreeNode treeNode =(DefaultMutableTreeNode) treePath.getPath()[1];
+					String userObject =(String) treeNode.getUserObject();
+					if(userObject.equals(locateValue))
+					{
+						nodeFound = true;
+					}
+					else
+					{
+						TreeNode childNodes = (TreeNode) treePath.getLastPathComponent();
+						Enumeration<?> children =  childNodes.children();
+						while(children.hasMoreElements())
+						{
+							DefaultMutableTreeNode child = (DefaultMutableTreeNode) children.nextElement();
+							userObject = (String) child.getUserObject();
+							if(userObject.equals(locateValue))
```

Comment:
user object value needs to compared with current node value and not the Locate value itself.

Reply:
Push the code for above changes. Pls review.